modules/cnv_editing_vm_web.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-editing-vm-web]]
+== Editing a virtual machine 
+
+You can edit some values of a virtual machine in the web console, either by editing the xref:cnv_editing_vm_yaml_web.adoc#cnv-editing-vm-yaml-web[YAML directly], or from the *Virtual Machine Overview* screen.
+
+When editing from the *Virtual Machine Overview* screen, the virtual machine must be *Off*.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines*  from the side menu.
+. Select a Virtual Machine.
+. Click *Edit* to make editable fields available. 
+. Changing the virtual machine *Flavor* to *Custom* provides additional fields for *CPU* and *Memory*.
```

Comment:
It would be good to mention here that you can pick only one of the "Custom" flavor or the flavor the VM has been created with. No other flavors are available on edit.
@alexxa FYI